src/components/NewVault.tsx

```diff
@@ -0,0 +1,34 @@
+import { useVaultStore } from 'store/vaults';
+
+const NewVault = ({ id }: { id: string }) => {
```

Comment:
please define `Props` type or interface. It's a good consistency for easy reading. 
```suggestion
const NewVault = ({ id }: Props) => {
```